odoo/addons/base/models/res_users.py

```diff
@@ -1391,6 +1466,20 @@ def unlink(self):
 class UsersView(models.Model):
     _inherit = 'res.users'
 
+    user_group_warning = fields.Text(compute="_compute_user_group_warning")
+
+    @api.depends('groups_id')
+    def _compute_user_group_warning(self):
```

Comment:
mhe this compute is bad from a performance pov; loop that call a method that loop on a search

the user is just used for the error message, can we remove the user from `_prepare_warning_for_group_inheritance` and from `_prepare_warnings_for_removed_groups`, call it only once for all groups and then parse the result in the loop ?
(and maybe add the user in the message in this compute)

what do you think ?